parsers/julia.c

```diff
@@ -1072,16 +1072,28 @@ static void parseFunction (lexerState *lexer, vString *scope, int parent_kind)
     if (lexer->cur_token != TOKEN_IDENTIFIER)
     {
         return;
-    } else if (lexer->cur_c == '.') {
+    }
+    else if (lexer->cur_c == '.')
+    {
         local_scope = vStringNewCopy(lexer->token_str);
         local_parent_kind = K_MODULE;
         advanceChar(lexer);
         advanceToken(lexer, true, false);
-    } else {
+    }
+    else
+    {
```

Comment:
I'm sorry that I accepted this style of change. I should not.

This is nothing to do with the original issue you wanted to fix.
Could you make a separate commit for the change of style?
In that case, I would like to use "Julia,cosmetic" or "Julia,style" as the prefix for the header of the commit log.

INSTEAD, I would like you to invite this project. So you can control your parser especially when we get a pull request for your parser. We can avoid the mistake I took.

Reply:
You can find "cosmetic" change with searching "cosmetic in git log output.